File: MODUL2/C/PRAK204_2410817310002_MuhammadRakhaAthallah.c
#include <stdio.h>

int main(){
    //variabel pertama
    float r, h;
    scanf("%f\n", &r);
    scanf("%f", &h);

float volume = 22.0/7.0 * r * r * h;
float luas = 2 * 22.0/7.0 * r * (r + h);
float keliling = 2 * 22.0/7.0 * r;

printf("Volume = %.2f\n", (volume));
printf("Luas = %.2f\n", (luas));
printf("Keliling = %.2f", (keliling));

//variabel kedua
float r1, h1;
scanf("%f", &r1);
scanf("%f", &h1);

float volume1 = 22.0/7.0 * r1 * r1 * h1;
float luas1 = 2 * 22.0/7.0 * r1 * (r1 + h1);
float keliling1 = 2 * 22.0/7.0 * r1;

printf("Volume = %.2f\n", (volume1));
printf("Luas = %.2f\n", (luas1));
printf("Keliling = %.2f", (keliling1));
return 0;
}

File: MODUL2/PYTHON/PRAK204_2410817310002_MuhammadRakhaAthallah.py
r = float(input())
h = float(input())

volume = 22.0/7.0 * r **2 * h
luas = 2 * 22.0/7.0 * r * (r + h)
keliling = 2 * 22.0/7.0 * r

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

r1, h1 = map(float, input().split())

volume1 = 22.0/7.0 * r1 **2 * h1
luas1 = 2 * 22.0/7.0 * r1 * (r1 + h1)
keliling1 = 2 * 22.0/7.0 * r1

print(f"Volume = {volume1:.2f}")
print(f"Luas = {luas1:.2f}")
print(f"Keliling = {keliling1:.2f}")